client/components/FormattingBar/FormattingBar.js

```diff
@@ -1,395 +1,287 @@
-import React, { Component } from 'react';
+import React, { useEffect, useRef, useState, useCallback } from 'react';
 import PropTypes from 'prop-types';
-import Icon from 'components/Icon/Icon';
-import {
-	OverflowList,
-	Boundary,
-	Button,
-	AnchorButton,
-	Classes,
-	Menu,
-	MenuItem,
-	Popover,
-	Position,
-	PopoverInteractionKind,
-} from '@blueprintjs/core';
-import { Overlay, DropdownButton } from 'components';
-import Controls from './Controls';
-import Media from './Media';
+import classNames from 'classnames';
+import { Toolbar, ToolbarItem, useToolbarState } from 'reakit';
+
+import { usePageContext } from 'containers/Pub/pubHooks';
+import { useRefMap } from '../../utils/useRefMap';
```

Comment:
Can simply use `from 'utils/useRefMap'`.

Reply:
Tell that to my editor!